roles/calico/templates/calico.service.j2

```diff
@@ -11,7 +11,7 @@ ExecStart=/usr/bin/docker run --net=host --privileged \
  -e WAIT_FOR_DATASTORE=true \
  -e FELIX_DEFAULTENDPOINTTOHOSTACTION=ACCEPT \
  -e CALICO_IPV4POOL_IPIP={{ calico_ipv4pool_ipip }} \
- -e CALICO_IPV4POOL_CIDR={{ calico_ipv4pool_cidr }} \
+ -e CALICO_IPV4POOL_CIDR={{ osm_cluster_network_cidr }} \
```

Comment:
If this variable isn't set you probably don't get the default since the default is probably buried in openshift_facts Python code. Will check tomorrow.